src/CascadeClassifierWrap.cc

```diff
@@ -74,28 +74,28 @@ class AsyncDetectMultiScale : public NanAsyncWorker {
   }
 
   void HandleOKCallback () {
-    NanScope();
+    Nan::HandleScope scope;
     //  this->matrix->Unref();
 
     Handle<Value> argv[2];
-    v8::Local<v8::Array> arr = NanNew<v8::Array>(this->res.size());
+    v8::Local<v8::Array> arr = Nan::New<v8::Array>(this->res.size());
 
     for(unsigned int i = 0; i < this->res.size(); i++ ){
-      v8::Local<v8::Object> x = NanNew<v8::Object>();
-      x->Set(NanNew("x"), NanNew<Number>(this->res[i].x));
-      x->Set(NanNew("y"), NanNew<Number>(this->res[i].y));
-      x->Set(NanNew("width"), NanNew<Number>(this->res[i].width));
-      x->Set(NanNew("height"), NanNew<Number>(this->res[i].height));
+      v8::Local<v8::Object> x = Nan::New<v8::Object>();
+      x->Set(Nan::New("x").ToLocalChecked(), Nan::New<Number>(this->res[i].x));
+      x->Set(Nan::New("y").ToLocalChecked(), Nan::New<Number>(this->res[i].y));
+      x->Set(Nan::New("width").ToLocalChecked(), Nan::New<Number>(this->res[i].width));
+      x->Set(Nan::New("height").ToLocalChecked(), Nan::New<Number>(this->res[i].height));
       arr->Set(i, x);
     }
 
-    argv[0] = NanNull();
+    argv[0] = Nan::Null();
     argv[1] = arr;
 
-    TryCatch try_catch;
+    Nan::TryCatch try_catch;
     callback->Call(2, argv);
```

Comment:
`argv` should be an array of type `v8::Local<v8::Value>` but here, it's an array of `v8::Handle<v8::Value>`.
[Nan::Callback](https://github.com/nodejs/nan/blob/master/doc/callback.md)